tests/test_service.py

```diff
@@ -10,32 +10,38 @@
 from unittest import TestCase
```

Comment:
@mdv5 - why don't you rename this test_service file to test_routes.py to make it consistent with our change back to routes?

Reply:
Changed the name and committed.

Reply:
amazing thanks so much! will review and merge